rotkehlchen/globaldb/updates.py

```diff
@@ -300,74 +305,219 @@ def _parse_full_insert(self, insert_text: str) -> AssetData:
             protocol=protocol,
```

Comment:
Also I guess `_parse_full_insert` is good to be renamed to show that it is only for assets